OffscreenCanvasAnimation.md

```diff
@@ -214,20 +215,38 @@ interface HTMLVideoElementFrameRequestData : FrameRequestData {
 }
 ```
 
-#### Issues
-* Does not support the use case of a continuously running task that produces frames repeatedly over time.  Not supporting this makes it inconvienient to port native application to the Web (e.g. via Emscripten) because the apps are often not deisigned for an asynchronous mode of operation.
+### OffscreenCanvas commit
+
+```
+interface OffscreenRenderingContext {
+  void commit();
+  readonly attribute OffscreenCanvas canvas;
+};
+
+interface OffscreenCanvasRenderingContext2D : OffscreenRenderingContext {};
+interface OffscreenWebGLRenderingContext : OffscreenRenderingContext, WebGLRenderingContext {};
+```
+
+#### The commit processing model
+
+When `commit()` is called, a copy of the frame is made and sent to the driver/display when appropriate. This function can return immediately or block. The UA can decide to queue a few frames before blocking to guarantee smoothness.
```

Comment:
Actually, we don't always make a copy. If it is a WebGL context with preserveDrawingBuffer = false, we transfer the frame instead.

Reply:
Missing: mechanism for throttling the frame rate